### ISSUE TITLE
Rambling Your Way to Clarity

### ISSUE THREAD

**Issue by farosato** [farosato/fabiorosatodotcom#5]
---
slug: rambling-to-clarity
subtitle: Giving yourself the freedom to ramble can be the key to reaching clarity
---
I'm taking part in Dickie Bush & Nicolas Cole's cohort-based course, [Ship 30 for 30](https://www.ship30for30.com/).
A great framework they recommend to consolidate your message is the `topic` + `for who` + `so that` formula.
Example: _Investing Tips For College Students To Graduate Without Any Debt_

It definitely acts as a forcing function to strengthen your message.
But if you don't already have a well-formed idea... it might end up paralyzing you.
**_WHO_** am I talking to? And **_WHAT_** precisely do I want to help them do?

So what if you don't have the clarity to use that formula yet?
Well, you either force yourself to come up with it before you start writing... or you could try something like what I'm doing today.
**Stream of consciousness**.

I get words flowing. I type whatever is going on in my mind as I think about the topic.
And I make damn sure I'm not filtering or editing in any way whatsoever at the moment.
**I'm just my mind transcriber**.
And the goal is to _fill the page_.
It's freeing.

IMPORTANT: **set a timer for 10 minutes when you do this**.
And when it goes off, you stop. This avoids rambling for too long and saves you time.

You then quickly reread what you wrote down and – sure enough – a clearer idea will have emerged.
**Writing brings clarity. Action over thinking.**

Then take that diamond in the rough and run with it.

To be respectful of your readers' time you'll have to get rid of the rambling and cut out probably 80% of your writing.
But the remaining 20% will be a condensed, information-rich piece you can safely try on your audience.

TL;DR: next time you're stuck on a blank page:

1. set a timer for 10 minutes and transcribe whatever goes through your mind as you think about the topic
2. identify the idea that emerged
3. cut out the rambling and condense it down to a solid piece.

Then just 🚢 it!
